index.html

```diff
@@ -9,36 +9,86 @@
     <link href="styles.css" rel="stylesheet" />
     <link href="one.css" rel="stylesheet" />
     <link href="one.js" rel="text/js" />
-    <title>One - Component Library</title>
+    <title>OneUI - Component Library</title>
 </head>
 
 <body>
+
+    <!--Navigation Bar-->
+    <nav class="nav">
+
+        <h1 class="navbar-brand">One UI</h1>
+        <ul class="navbar-nav">
+            <li class="nav-item"><a class="nav-link" href=#>Home</a></li>
+            <li class="nav-item"><a class="nav-link" href=#><strong>Docs</strong></a></li>
+        </ul>
+        <div class="version-number">
+            <span>V1.0</span>
+        </div>
+
+    </nav>
+
     <div class="main-container">
         <aside>
             <h1>Contents</h1>
             <div class="aside-container">
-                <a href="#avatars" class="side-item">Avatar</a>
                 <a href="#alerts" class="side-item">Alert</a>
+                <a href="#avatars" class="side-item">Avatar</a>
                 <a href="#badges" class="side-item">Badge</a>
                 <a href="#buttons" class="side-item">Button</a>
                 <a href="#cards" class="side-item">Card</a>
-                <a href="#images" class="side-item">Image</a>
+                <a href="#grids" class="side-item">Grid</a>
+                <a href="#images" class="side-item">Images</a>
                 <a href="#inputs" class="side-item">Input</a>
                 <a href="#lists" class="side-item">List</a>
-                <a href="#navigation" class="side-item">Navigation</a>
                 <a href="#modals" class="side-item">Modal</a>
+                <a href="#navigation" class="side-item">Navigation</a>
                 <a href="#ratings" class="side-item">Rating</a>
                 <a href="#sliders" class="side-item">Slider</a>
-                <a href="#grids" class="side-item">Grid</a>
                 <a href="#utilities" class="side-item">Text-Utilities</a>
                 <a href="#toasts" class="side-item">Toast</a>
```

Comment:
Hi Mohan,
Can we use a list tag here? Because that would be semantic I guess.

Reply:
done